### PR TITLE
Feature/Adding-new-reporting-table-for-on-ART--sentinel-event

### PR DESCRIPTION
**Pull request by nobert-mumo**
This PR introduces a new case surveillance reporting table to answer onART numbers longitudinally.

### CODE REVIEW

**Review by Marymary-dev — APPROVED** [review]

File: Scripts/REPORTING/load_cs_aggregate_onART_sentinel_event.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
+IF OBJECT_ID(N'[HIVCaseSurveillance].[dbo].[CsAggegateOnARTSentinelEvent]', N'U') IS NOT NULL 
+	drop TABLE [HIVCaseSurveillance].[dbo].[CsAggegateOnARTSentinelEvent]
```
Comment (cbrianbet) [inline Scripts/REPORTING/load_cs_aggregate_onART_sentinel_event.sql:2]: Just a nitpick this should be **Aggregate** I see it created 2 tables on .83

**Review by Marymary-dev — CHANGES_REQUESTED** [review]
@nobert-mumo  the numbers for onART should be the numbers on ART AsofDate for the selected Cohort

File: Scripts/REPORTING/load_cs_aggregate_onART_sentinel_event.sql (hunk 1 above)
Comment (nobert-mumo) [reply Scripts/REPORTING/load_cs_aggregate_onART_sentinel_event.sql]: @cbrianbet thanks. This is updated. I will drop the outdated table in the DB.

**Comment by nobert-mumo** [comment]
> @nobert-mumo the numbers for onART should be the numbers on ART AsofDate for the selected Cohort

@Marymary-dev just confirming this means we should show cummulative As Of ?

**Comment by Marymary-dev** [comment]
yes, sorry had not seen this

**Comment by nobert-mumo** [comment]
> yes, sorry had not seen this

Hi @Marymary-dev this is updated we know have running totals for Started on ART Clients for the last 12 months from the reporting period

**Review by Marymary-dev — APPROVED** [review]
Will borrow a lot from the cursor object. Just thinking aloud if this table can be reused by many other tables

**Comment by nobert-mumo** [comment]
> Will borrow a lot from the cursor object. Just thinking aloud if this table can be reused by many other tables

Indeed can be reused. Will now merge

**Pull request merged**